bayes_opt/util.py

```diff
@@ -21,13 +25,13 @@ def load_logs(optimizer, logs):
     Returns
     -------
     The optimizer with the state loaded.
-    
+
     """
     if isinstance(logs, str):
         logs = [logs]
 
     for log in logs:
-        with open(log, "r") as j:
+        with open(log) as j:  # noqa: PTH123
```

Comment:
Would also be fine with using Pathlib here.

Reply:
I'll fix it soon.